src/Cli/dotnet/SlnFileExtensions.cs

```diff
@@ -255,6 +293,19 @@ private static void AddSolutionFolders(this SlnFile slnFile, SlnProject slnProje
             }
         }
 
+        private static string GetSolutionFolderWithDuplicateProject(SlnFile slnFile, SlnProject slnProject, IList<string> solutionFolders, SlnSection nestedProjectsSection)
```

Comment:
I do really like this function. It has a complicated task but it's very clear and concise code for what it is 🎆